src/main/resources/application.properties

```diff
@@ -13,4 +13,4 @@ npg.uri=${NPG_URI}
 npg.readTimeout=${NPG_READ_TIMEOUT}
 npg.connectionTimeout=${NPG_CONNECTION_TIMEOUT}
 npg.client.maxInMemory=${SPRING_MAX_IN_MEM_SIZE}
-npg.client.key=${NPG_KEY}
\ No newline at end of file
+npg.client.key=${NPG_API_KEY}
```

Comment:
```suggestion
npg.client.apiKey=${NPG_API_KEY}
```

What do you think?

Reply:
Yeah, even better 🙂

Reply:
Done in [bf3aa28](https://github.com/pagopa/pagopa-ecommerce-payment-methods-service/pull/82/commits/bf3aa282fd24e75af9cb7607d5dbab8033acc55e)